reactive/docker.py

```diff
@@ -180,7 +185,7 @@ def install():
         hookenv.log('Unknown runtime {}'.format(runtime))
         return False
 
-    validate_config()
+    validate_config(config())
```

Comment:
I think we want to use `check_for_juju_https_proxy` here, too. As it is now, it looks like this charm will initially configure and start the docker service without including the juju-proxy vars, until a future `recycle_daemon` corrects it.

Reply:
Makes sense